### PR TITLE
Document supported versions and test minimum

### LINKED ISSUE #1100: Properly document minimum version and test it
coreutils doesn't build on nix due to https://github.com/uutils/coreutils/issues/1064. Nix only supports Rust 1.17.0 currently: (https://nixos.org/nixos/packages.html, search for "rustc")

It would be less confusing to newcomers if `coreutils` would document the current minimum version of rustc it currently needs and tests against that.

### PR DESCRIPTION
**Pull request by skade**
Fixes #1100

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ are either old, abandoned, hosted on CVS, written in platform-specific C, etc.
 Rust provides a good, platform-agnostic way of writing systems utils that are easy
 to compile anywhere, and this is as good a way as any to try and learn it.
 
+Rust Version
+------------
+
+coreutils supports and is tested on version `1.20.0` and above (including beta and nightly builds) of `rustc`.
```
Comment (Arcterus) [inline README.md]: `coreutils` should probably be changed to `uutils` to keep consistent with the rest of the README.

Comment (skade) [reply README.md]: Changed.

Should the same be done here? https://github.com/uutils/coreutils#contribute

**Comment by skade** [comment]
On Windows: that probably makes sense, but can be done at a later point. Does it make sense to indicate that in the README?

File: README.md (hunk 1 above)
Comment (Arcterus) [reply README.md]: Ah, I never noticed that.  Yes, please change that one as well.

**Comment by Arcterus** [comment]
I think so, as currently there are no guarantees that the code works for Windows on stable unless you test it yourself.

**Comment by Arcterus** [comment]
Thanks, hopefully this will help people get started with the project.

**Pull request merged**